lib/mix/lib/mix/utils.ex

```diff
@@ -15,28 +15,33 @@ defmodule Mix.Utils do
   stored there.
   """
   def mix_home do
-    mix_home_xdg_lookup("XDG_DATA_HOME")
+    mix_home_xdg_lookup(:user_data)
   end
 
   @doc """
   Gets possible location of global Mix configuration.
 
   Possible locations:
 
-   * `~/.mix`
-   * `MIX_HOME`
-   * `XDG_CONFIG_HOME/mix`
+     * `XDG_CONFIG_HOME/mix`
+     * `MIX_HOME`
+     * `~/.mix`
```

Comment:
Although the documentation does not talk about the priority of those locations, I would order them by it:

```suggestion
     * `MIX_HOME`
     * `XDG_CONFIG_HOME/mix` (if `MIX_XDG` is set)
     * `~/.mix`
```

In addition, I would mention that `XDG_CONFIG_HOME` only works if `MIX_XDG` is set because it might not be obvious that you need to set that environment variable (for example if you land directly in the documentation of this function). The same applies to `mix_home/0` 🙂

Reply:
Ups! Nevermind. This module is not publicly documented.